common/app/common/dfp/DfpAgent.scala

```diff
@@ -98,11 +98,11 @@ object DfpAgent
         maybeLineItems getOrElse Nil
       }
 
-      lineItemAgent sendOff { oldData =>
+      Future(lineItemAgent send { oldData =>
```

Comment:
The difference of this approach is that sendOff enforces updates in order. ```Future(send(...))``` doesn't